contentcuration/contentcuration/static/js/edit_channel/channel_list/vuex/actions.js

```diff
@@ -81,6 +83,7 @@ export function deleteChannel(context, channelID) {
     new Models.ChannelModel(channel).save(
       { deleted: true },
       {
+        patch: true,
```

Comment:
This fixes channel deletion as we just want to change the 'deleted' field and nothing else